### ISSUE TITLE
Fake News Detection

### ISSUE THREAD

**Issue by nk-droid** [abhisheks008/DL-Simplified#253]
Fake news detection system to detect if a news is real or fake.

**Comment by Sagar201003** [comment]
I can work on this project 
FullName: Sagar Santoshkumar Shukla
Github Profile Link: https://github.com/Sagar201003
Email ID: [shuklasagar201003@gmail.com](mailto:shuklasagar201003@gmail.com)

Approach for this Project:
I will perform EDA , feature selection , PCA , Data augmentation . I will use Pycaret to get the best model .
Implement the models using TensorFlow or PyTorch.

What is your participant role? -SSOC contributor

**Comment by abhisheks008** [comment]
> I can work on this project FullName: Sagar Santoshkumar Shukla Github Profile Link: https://github.com/Sagar201003 Email ID: [shuklasagar201003@gmail.com](mailto:shuklasagar201003@gmail.com)
> 
> Approach for this Project: I will perform EDA , feature selection , PCA , Data augmentation . I will use Pycaret to get the best model . Implement the models using TensorFlow or PyTorch.
> 
> What is your participant role? -SSOC contributor

Please try to find out a different issue as the contributor himself has opened this issue. So this will be assigned to him after evaluating all the parameters.

**Comment by nk-droid** [comment]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** :  Fake News Detection Model</br>
:red_circle: **Aim** : Create and compare some models to detect if a news is real or not.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/clmentbisaillon/fake-and-real-news-dataset</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Nidhish Kumar
- GitHub Profile Link :  https://github.com/nk-droid
- Email ID : nidhish.22k@gmail.com
- Participant ID (if applicable):
- Approach for this Project : First, I'll make simple ML and DL models and then compare their performance.
- What is your participant role? SSoC'23 Contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Complete the assigned issue first, after that this issue will be assigned to you.
@nk-droid

**Comment by nk-droid** [comment]
@abhisheks008 I've completed the assigned task. Could you please merge it and then assign me this project?

**Comment by abhisheks008** [comment]
Issue assigned to you @nk-droid. You can start working on it.

**Comment by maitri-vv** [comment]
@abhisheks008 @nk-droid i would too like to work on this issue.

**Comment by abhisheks008** [comment]
This issue is already assigned @maitri-vv

**Comment by YashSachan2** [comment]
Hi @abhisheks008 Please assign this issue to me.I have worked on a similar case of spam classifer and fraud insurance claims detection problem before.

**Comment by abhisheks008** [comment]
Hi @YashSachan2 can you please share your approach for solving this issue?

**Comment by YashSachan2** [comment]
I intend to create a batches of dataset with 70% false dataset and 30% true dataset so as to emphasize more on correctly identifying wrong articles ,then doing some feature engineering like length of article,positive words count,negative words count etc.Then after encoding using word2vec then comparison and hypertuning among models like XG Boost,Cat boost ,NN having dual RNN and LSTM layers along with embedding layer,Logistic Regression.

**Comment by abhisheks008** [comment]
Looks good to me. Issue assigned to you. @YashSachan2

**Comment by abhisheks008** [comment]
Hi @YashSachan2 any update on this issue?

**Comment by YashSachan2** [comment]
@abhisheks008 I am working on it. I will submit pr by today

**Comment by YashSachan2** [comment]
@abhisheks008 I have made the pr for this issue.Please look into it and assign me the points

**Comment by YashSachan2** [comment]
@abhisheks008 I have made a new pr making the changes you asked for.Please look into it and assign me the points.

**Comment by abhisheks008** [comment]
> @abhisheks008 I have made a new pr making the changes you asked for.Please look into it and assign me the points.

Hi @YashSachan2 I have requested some changes there. Once the PR is approved and merged, the points will be assigned accordingly. You don't need to make me remember every single time :)
